lib/ssl/src/tls_sender.erl

```diff
@@ -582,7 +582,7 @@ update_bytes_sent(_, #data{static = #static{bytes_sent = Sent} = Static} = State
 key_update_at(Version, #{security_parameters :=
                              #security_parameters{
                                 bulk_cipher_algorithm = CipherAlgo}}, KeyUpdateAt)
-  when Version >= {3,4} ->
+  when Version >= ?'TLS-1.3' ->
```

Comment:
Maybe we should test for TLS-1.3 here, we do not know when and if TLS-1.4 will be equivalent! Same comment for the lines below.